cmake/projects/ZeroMQ/hunter.cmake

```diff
@@ -15,11 +16,12 @@ hunter_add_version(
   VERSION
   "4.1.4"
   URL
-  "https://github.com/hunter-packages/zeromq4-1/archive/v4.1.4-hunter-p1.tar.gz"
+  "https://github.com/hunter-packages/zeromq4-1/archive/v4.1.4-hunter-p2.tar.gz"
```

Comment:
You can add new version instead of rewriting. It will be surprise for users who relies on some features of old version.